### PR TITLE
Add product template to the doc loader.

### PR DESCRIPTION
**Pull request by ritalrw**

### CODE REVIEW

**Comment by ritalrw** [comment]
@pavan-couchbase please review!

File: src/main/java/couchbase/test/val/Product.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        JsonObject jsonObject = JsonObject.create();
+        this.random.setSeed((key).hashCode());
+        if(this.ws.mutated > 0){
+            this.random.setSeed((key+Integer.toString(this.ws.mutated)).hashCode());
+        }
+        int index = this.random.nextInt(4096);
+        jsonObject.put("id", "P" + key);
+        jsonObject.put("product_name", faker.name().firstName());
+        jsonObject.put("product_link", faker.internet().url());
+        jsonObject.put("product_features", this.buildProductFeatures(1 + this.random.nextInt(3)));
```
Comment (pavan-couchbase) [inline src/main/java/couchbase/test/val/Product.java:96]: Looks good to me, but a couple questions - 

~~seller_verified is something you've added? I don't see it in the existing template~~ .

Should we also have some docs with 0 length arrays for product links/ reviews etc? Just for the sake of variety.

**Review by pavan-couchbase — CHANGES_REQUESTED** [review]

File: src/main/java/couchbase/test/val/Product.java (hunk 1 above)
Comment (pavan-couchbase) [reply src/main/java/couchbase/test/val/Product.java:96]: I was looking at an older version of the template. seller_verified is part of the new one. I've struck it from my  comment.

Comment (ritalrw) [reply src/main/java/couchbase/test/val/Product.java:96]: Product link is a string not an array and for reviews, products with 0 reviews are already considered.

**Review by pavan-couchbase — APPROVED** [review]

**Pull request merged**